README_ENTRYPOINT.md

```diff
@@ -0,0 +1,17 @@
+There is no file shebang for python that works on all platforms (#71818).
+To minimize breakage we have chosen to make this work when `env` is in path, or when using python/python2/python3 directly.
+Unfortunately, this breaks users using the `py` wrapper on Windows, where env isn't supported outside of MingW shells.
```

Comment:
```suggestion
To minimize breakage we have chosen to make this work when `bash` is available, or when using python/python2/python3 directly.
Unfortunately, this breaks users using the `py` wrapper on Windows, where `bash` isn't supported outside of MingW shells.
Existing versions of `py` will see the bash shebang line and try to respect it and interpret the file with `bash`.
```

Reply:
I mean, technically you need both `env` and `bash`; I can't imagine having one without the other (maybe there are tiny docker containers without bash? we can change to using `sh` if needed).

I guess saying bash is more helpful to end users though? not sure that people are reading anything other than the instructions

Reply:
From the Windows perspective at least, `env` isn't needed, since `py` emulates it and searches in `path`.